.gitlab-ci.yml

```diff
@@ -245,6 +245,32 @@ build:base+async:
     COQ_EXTRA_CONF: "-native-compiler yes -coqide opt"
     COQUSERFLAGS: "-async-proofs on"
 
+quick:build+validate:
+  stage: test
+  script:
+    - set -e
+
+    - echo 'start:coq.config'
+    - ./configure -local
+    - echo 'end:coq.config'
+
+    - echo 'start:coqc.quick'
+    - COQUSERFLAGS="-quick" make -j "$NJOBS"
+    - echo 'end:coqc.quick'
+
+    - echo 'start:coqc.vio2vo'
+    - bin/coqc -boot -schedule-vio2vo "$NJOBS" `find . -name \*.vio`
```

Comment:
All this custom logic doesn't belong into the CI scripts [think for example we would like to call from Azure]

Reply:
We want to run -quick from azure?

Reply:
Do we support `-quick` on Windows?

Reply:
> All this custom logic doesn't belong into the CI scripts

Do we support validate on Azure? (this line is very close to the one of validate)

Reply:
Which dependency path? I'm sorry but I don't understand. What would be the problem on Azure in the first place?

Reply:
Ideally the first target would be produced by the makefile just by depending on `VOFILES:.vo:=.vio`, then, the second step should also be produced from a make rule that knows how to do `.vio -> .vo`.